be/src/exec/pipeline/hashjoin/spillable_hash_join_build_operator.cpp

```diff
@@ -110,7 +110,7 @@ Status SpillableHashJoinBuildOperator::set_finishing(RuntimeState* state) {
                 state, *io_executor, TRACKER_WITH_SPILLER_GUARD(state, spiller));
     };
 
-    publish_runtime_filters(state);
+    RETURN_IF_ERROR(publish_runtime_filters(state));
```

Comment:
publish runtime filter error is not a critical error. we could ignore it. SpillProcessTasksBuilder shouldn't be ignored

Reply:
add a warning?

Reply:
fixed, change to to WARN_IF_ERROR